packages/user/src/views/Login.tsx

```diff
@@ -109,6 +117,18 @@ const Login: React.FC<IProperties> = ({
     );
   };
 
+  const showReminder = () => {
+    return (
+      <div className="reminder">
+        {getPath() && getPath()?.length && (
```

Comment:
use `useMemo` to compute the redirection path instead of calling getPath everytime we need it.